src/org/opendatakit/briefcase/reused/http/CommonsHttp.java

```diff
@@ -147,6 +151,16 @@ public void unsetProxy() {
       throw new HttpException("The connection has timed out", e);
     } catch (IOException e) {
       throw new UncheckedIOException(e);
+    } finally {
+      try {
+        UncheckedFiles.closeInputStream(request.getBody());
+      } catch (final BriefcaseException exception) {
+        LOG.error("Error: Attempted to close Request.body InputStream, but failed", exception);
```

Comment:
Not sure if this is something actionable or that would help us help a user having trouble with Briefcase. @timkoon, do you think it would be useful to have this on the log?

Reply:
True, it may not be actionable, but in the least, it would give insight into any system issues. I think there's value for the log to be there even though it may be rare that it will trigger.

With the alternative, I don't think there's value if the catch statement does nothing.